robotbase/src/main/java/com/team2357/frc2022/subsystems/TurretSubsystem.java

```diff
@@ -91,22 +92,26 @@ public void resetHeading() {
         m_turretMotor.getEncoder().setPosition(0);
     }
 
-    public double getTurretRotations() {
-        return m_turretMotor.getEncoder().getPosition();
+    public double getTurretDegrees() {
+        return m_turretMotor.getEncoder().getPosition() / m_config.m_rotationsPerDegree;
```

Comment:
I'm wondering if we should use a "clicks per degree" integer variable instead of motor rotations.

Reply:
We can add a factor of 42 onto the the spark max so the getPosition() method actually returns encoder clicks, but I do not think this is necessary since it returns rotations by default.